packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -358,10 +374,20 @@ const AccountSwitcher = props => {
         let total = vrtc_currency !== props.obj_total_balance.currency ? vrtc_balance * exchanged_rate : vrtc_balance;
 
         if (Array.isArray(props.mt5_login_list)) {
-            total += mt5_demo_total.balance;
+            const mt5_demo_currency = props.mt5_login_list.find(account => isDemo(account))?.currency;
+
+            total +=
+                mt5_demo_currency !== props.obj_total_balance.currency
+                    ? mt5_demo_total.balance * exchanged_rate_mt5_dxtrade.mt5_demo
+                    : mt5_demo_total.balance;
         }
         if (Array.isArray(props.dxtrade_accounts_list)) {
-            total += dxtrade_demo_total.balance;
+            const dxtrade_demo_currency = props.dxtrade_accounts_list.find(account => isDemo(account))?.currency;
+
+            total +=
+                dxtrade_demo_currency !== props.obj_total_balance.currency
+                    ? dxtrade_demo_total.balance * exchanged_rate_mt5_dxtrade.dxtrade
```

Comment:
`? dxtrade_demo_total.balance * exchanged_rate_mt5_dxtrade.dxtrade_demo`

Reply:
thanks, will change it